frontend/packages/yki/src/components/layouts/SessionStateHeader.tsx

```diff
@@ -39,12 +42,49 @@ const generateLogoutURL = () => {
 };
 
 export const SessionStateHeader: FC = () => {
+  const location = useLocation();
   const translateCommon = useCommonTranslation();
+  const userOpenRegistrations = useAppSelector(userOpenRegistrationsSelector);
   const session = useAppSelector(sessionSelector).loggedInSession;
-  if (!session || session.identity === null) return <></>;
+  const dispatch = useAppDispatch();
+  const hasNullIdentity = !session || session.identity === null;
```

Comment:
Jäin pohtimaan olisiko luontevampaa kääntää tämän booleanin merkitys ja nimi; esim. siis `const hasSessionIdentity = ...` tai jopa `const isAuthenticated = ...`.

Ei sinänsä iso asia, voi jäädä näinkin.